src/main/java/cz/it4i/fiji/datastore/security/SecurityRegistry.java

```diff
@@ -34,7 +32,66 @@ class SecurityRegistry {
 
 	private Map<String, OAuthServer> servers;
 
+	private List<OAuthGroup> groups;
+
 	private Boolean securityDisabled;
+	private Boolean staticSecurity;
+
+
+	public List<String> getServerAliases()
+	{
+		List<String> names=new ArrayList<>();
+		for(int i=0;i<servers.size();i++)
+		{
+			names.add(servers.get(i).getName());
+		}
+		return names;
+	}
+	public void refreshServers()
+	{
+		if (isSecurityDisabled()) {
+			return;
+		}
+		List<String[]> list_server= new ArrayList<>();
+		LinkedList<OAuthServer> temp = new LinkedList<>();
+
+		if(isStaticSercuritySet()) {
+			String[] serversTokens = getProperty(SECURITY_SERVERS).split(";");
```

Comment:
Use security properties for an initialisation of an empty database. After that ignore it. For reading data use only db.